drivers/serial/uart_mspm0g3xxx.c

```diff
@@ -99,21 +99,24 @@ static void uart_mspm0g3xxx_irq_tx_enable(const struct device *dev)
 {
 	const struct uart_mspm0g3xxx_config *config = dev->config;
 
-	DL_UART_enableInterrupt(config->regs, DL_UART_INTERRUPT_TX);
+	DL_UART_enableInterrupt(config->regs, DL_UART_INTERRUPT_TX | DL_UART_INTERRUPT_EOT_DONE);
 }
 
 static void uart_mspm0g3xxx_irq_tx_disable(const struct device *dev)
 {
 	const struct uart_mspm0g3xxx_config *config = dev->config;
 
-	DL_UART_disableInterrupt(config->regs, DL_UART_INTERRUPT_TX);
+	DL_UART_disableInterrupt(config->regs, DL_UART_INTERRUPT_TX | DL_UART_INTERRUPT_EOT_DONE);
```

Comment:
Seems like the combination of `DL_UART_INTERRUPT_TX | DL_UART_INTERRUPT_EOT_DONE` is reused. You could define it to clean up slightly and also potentially prevent errors in forgetting one in some place

Reply:
Funny, I started doing that but it felt odd due to RX only having one property :laughing: 
I'll do it anyways then :)